src/patternfly/components/Button/styles.scss

```diff
@@ -290,6 +312,66 @@
     }
   }
 
+  // Link and Icon Buttons
+  &.pf-m-link,
+  &.pf-m-icon {
+    background-color: transparent;
+    box-shadow: none;
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    &:focus,
+    &.pf-m-focus {
+      background-color: transparent;
+      box-shadow: none;
+    }
+  }
+
+  // Link Buttons
+  &.pf-m-link {
+    padding: var(--pf-c-button--m-link--PaddingTop) var(--pf-c-button--m-link--PaddingRight) var(--pf-c-button--m-link--PaddingBottom) var(--pf-c-button--m-link--PaddingLeft);
```

Comment:
I saw a comment related to this, but can't find it now, so I apologize if this was already resolved and I'm kicking it back up again... 

Would we want links to have the same padding that buttons have by default, and then include another modifier class for cases where you want the link text to be left-/right-aligned with elements above or below it?

Reply:
lets only change the left and right padding on both actions and links